test/common/stats/thread_local_store_test.cc

```diff
@@ -1149,6 +1149,36 @@ TEST_F(StatsMatcherTLSTest, RejectPrefixNoDot) {
   EXPECT_MEMORY_LE(mem_consumed, 3500000);
 }
 
+TEST_F(StatsMatcherTLSTest, DoNotRejectHiddenRegex) {
+  InSequence s;
```

Comment:
Why is this needed? The above test in the same test-fixture does not. I think this test might not have any mocks.